packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -130,7 +130,7 @@ class SwipeableDrawer extends React.Component {
       drawerStyle.transition = transition;
     }
 
-    if (!this.props.disableBackdropTransition && !this.props.hideBackdrop) {
+    if (this.backdropRef && !this.props.disableBackdropTransition && !this.props.hideBackdrop) {
```

Comment:
`this.backdropRef` could be falsey if the underlying `<Modal />` component is closed whilst the transitioning state is "entering" - This is because `this.setState({ maybeSwiping: false })` is called, re-rendering the component and changing the Drawer's `open` prop to change from `true` to `open` (which is equal to `false`), which leads to the `<Modal />` children unmounting due to the defaulted `keepMounted: false` prop.

Reply:
It's no longer needed.